src/komponenter/ContextProvider.tsx

```diff
@@ -24,6 +24,7 @@ export interface Context {
     settPermitteringInnhold: SettPermitteringInnhold;
     setSideSistOppdatert: SettSideSistOppdatert;
     dagensDato: Date;
+    tidligsteDatoAGP2: Date;
```

Comment:
Jeg ser for meg at denne variabelen kan forveksles med andre variabler vi bruker i utregningen - hva med å kalle den for f.eks. `agp2Innføringsdato`?